content/.htaccess

```diff
@@ -0,0 +1,375 @@
+ Redirect 301 /questionable-authorship/of-the-old-and-new-testament.html /works/questionable-authorship/of-the-old-and-new-testament.html
+ Redirect 301 /questionable-authorship/of-the-sabbath-day-in-Connecticut.html /works/questionable-authorship/of-the-sabbath-day-in-Connecticut.html
+ Redirect 301 /questionable-authorship/plan-of-a-declaration.html /works/questionable-authorship/plan-of-a-declaration.html
+ Redirect 301 /questionable-authorship/reflections-on-titles.html /works/questionable-authorship/reflections-on-titles.html
+ Redirect 301 /questionable-authorship/reflections-on-unhappy-marriages.html /works/questionable-authorship/reflections-on-unhappy-marriages.html
+ Redirect 301 /questionable-authorship/reflections-on-the-life-and-death-of-lord-clive.html /works/questionable-authorship/reflections-on-the-life-and-death-of-lord-clive.html
+ Redirect 301 /questionable-authorship/retreat-across-the-delaware.html /works/questionable-authorship/retreat-across-the-delaware.html
+ Redirect 301 /questionable-authorship/the-american-philosophical-society.html /works/questionable-authorship/the-american-philosophical-society.html
+ Redirect 301 /questionable-authorship/the-dream-interpreted.html /works/questionable-authorship/the-dream-interpreted.html
+ Redirect 301 /questionable-authorship/the-society-for-political-inquiries.html /works/questionable-authorship/the-society-for-political-inquiries.html
+ Redirect 301 /questionable-authorship/thoughts-on-defensive-war.html /works/questionable-authorship/thoughts-on-defensive-war.html
+ Redirect 301 /questionable-authorship/to-the-members-of-the-society.html /works/questionable-authorship/to-the-members-of-the-society.html
+ Redirect 301 /questionable-authorship/to-the-people-of-connecticut.html /works/questionable-authorship/to-the-people-of-connecticut.html
+ Redirect 301 /questionable-authorship/useful-and-entertaining-hints.html /works/questionable-authorship/useful-and-entertaining-hints.html
+ Redirect 301 /recently-discovered/a-fifth-forester-letter.html /works/recently-discovered/a-fifth-forester-letter.html
+ Redirect 301 /recently-discovered/attack-on-paper-money-laws.html /works/recently-discovered/attack-on-paper-money-laws.html
+ Redirect 301 /recently-discovered/four-letters-on-interesting-subjects.html /works/recently-discovered/four-letters-on-interesting-subjects.html
+ Redirect 301 /recently-discovered/nonsense-from-new-york.html /works/recently-discovered/nonsense-from-new-york.html
+ Redirect 301 /recently-discovered/on-the-affairs-of-the-state.html /works/recently-discovered/on-the-affairs-of-the-state.html
+ Redirect 301 /recently-discovered/response-to-observations-on-the-american-revolution.html /works/recently-discovered/response-to-observations-on-the-american-revolution.html
+ Redirect 301 /recently-discovered/response-to-an-accusation-of-bribery.html /works/recently-discovered/response-to-an-accusation-of-bribery.html
```

Comment:
```suggestion
RewriteEngine On

RewriteRule ^major-works(/.*)?$ /works/major-works$1 [L,NC,NE,R=301]
RewriteRule ^essays(/.*)?$ /works/essays$1 [L,NC,NE,R=301]
RewriteRule ^letters(/.*)?$ /works/letters$1 [L,NC,NE,R=301]
RewriteRule ^questionable-authorship(/.*)?$ /works/questionable-authorship$1 [L,NC,NE,R=301]
RewriteRule ^recently-discovered(/.*)?$ /works/recently-discovered$1 [L,NC,NE,R=301]
RewriteRule ^aboutpaine(/.*)?$ /pages/resources$1 [L,NC,NE,R=301]

RewriteRule ^membership.html /pages/membership.html
RewriteRule ^pages/the-official-collected-works-of-thomas-paine-project.html /pages/collected-works-project.html

 ```

Reply:
I think the above suggested change should handle everything we need for redirects. If it looks good to you, go ahead and commit the changes into this branch and then we can merge to update the live site.